ASP.NET Core/gulpfile.js

```diff
@@ -0,0 +1,62 @@
+/// <binding Clean='clean' />
+var gulp = require("gulp"),
+    rimraf = require("rimraf"),
+    concat = require("gulp-concat");
+
+var paths = {
+    webroot: "./wwwroot/"
+};
+
+paths.concatJsDest = paths.webroot + "js/vendor.js";
+paths.concatCssDest = paths.webroot + "css/vendor.css";
+paths.fonts = paths.webroot + "css/fonts";
+paths.icons = paths.webroot + "css/icons";
+
+let scripts = ["node_modules/jquery/dist/jquery.js",
```

Comment:
We can create a couple of such script bundles here. The main idea is to have the resulting bundle with all scripts, with minimal scripts, with scripts for export and scripts for HTML editor.

Reply:
I added more bundles in the last PR